bazel/amalgamate.py

```diff
@@ -35,95 +35,97 @@ def parse_include(line):
   match = INCLUDE_RE.match(line)
   return match.groups()[0] if match else None
 
+def is_core_upb_header(fname):
```

Comment:
This doesn't look like it's used anywhere?

Reply:
Done.